### PR TITLE
Share styles for common elements across blog and documentation. 

### LINKED ISSUE #561: design: markdown bullet points are not aligned
bullet points are not aligned with the rest of the text

_Originally posted by @ChristinFrohne in https://github.com/gitpod-io/website/pull/548#discussion_r644571327_

See

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #561 

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/568"></a>

### CODE REVIEW

**Comment by nisarhassan12** [comment]
@mikenikles As suggested with your upvote on slack([internal)](https://gitpod.slack.com/archives/C01KGM9ETU6/p1623061399067300) I have added the external link arrow to the external links on the docs but now we have an issue with images that are wrapped within external links they have an arrow appearing next to them. I can't think of a possible CSS only solution here what do you think about using some JS here to add a class to such links who have an image as the only child which can then be targetted with CSS ? Thanks
image

**Comment by mikenikles** [comment]
Regardless of where we keep the CSS to indicate external links, we have that issue. Adding a CSS class to links with an image only is potentially an option if we create a remark plugin similar to [the two we already have](https://github.com/gitpod-io/website/tree/main/src/utils).

We'd have to look for anchors, check the children and if it's a single image child, add the CSS class. Do you want to give that a go?

File: src/utils/remark-link-with-image-as-only-child.cjs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const visit = require("unist-util-visit");
```
Comment (mikenikles) [inline src/utils/remark-link-with-image-as-only-child.cjs:1]: Great work!

**Review by mikenikles — APPROVED** [review]
LGTM

**Pull request merged**